sncosmo/builtins.py

```diff
@@ -66,31 +75,51 @@ def get_rootdir():
 def load_bandpass_bessell(pkg_data_name, name=None):
     """Bessell bandpasses have (1/energy) transmission units."""
     fname = get_pkg_data_filename(pkg_data_name)
-    return read_bandpass(fname, wave_unit=u.AA, trans_unit=u.erg**-1,
-                         normalize=True, name=name)
+    try:
+        return read_bandpass(fname, wave_unit=u.AA, trans_unit=u.erg**-1,
+                             normalize=True, name=name)
+    except:
+        rm(fname)
+        raise
 
 
 def load_bandpass_remote_aa(relpath, name=None):
     abspath = DATADIR.abspath(relpath)
-    return read_bandpass(abspath, wave_unit=u.AA,
-                         trim_level=BANDPASS_TRIM_LEVEL, name=name)
+    try:
+        return read_bandpass(abspath, wave_unit=u.AA,
+                             trim_level=BANDPASS_TRIM_LEVEL, name=name)
+    except:
```

Comment:
Is there a way to not copy and past code? As we add bandpass options, I would rather not require future developers to remember to write a 5 line return statement.